build.cmd

```diff
@@ -334,8 +343,22 @@ if '%TEST_PORTABLE_COREUNIT%' == '1' (
 )
 
 if '%BUILD_VS%' == '1' (
-    %_msbuildexe% %msbuildflags% VisualFSharp.sln /p:Configuration=%BUILD_CONFIG%
```

Comment:
It's all going away soon, we will be using a modified this:

https://github.com/Microsoft/visualfsharp/blob/master/microbuild-everything.proj